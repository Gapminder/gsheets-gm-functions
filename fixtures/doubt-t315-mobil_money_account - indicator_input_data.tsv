geo	name	year	value
lbr	Liberia	2014	..
mwi	Malawi	2014	3.844100952
zaf	South Africa	2014	14.42818069
chl	Chile	2014	3.789850712
ben	Benin	2014	2.022727013
tur	Turkey	2014	0.808199763
cod	Congo, Dem. Rep.	2014	9.20866394
tcd	Chad	2014	5.750106812
cmr	Cameroon	2014	1.796604872
gin	Guinea	2014	1.473285675
hti	Haiti	2014	3.782689571
mdg	Madagascar	2014	4.35601902
sle	Sierra Leone	2014	4.45238781
ven	Venezuela	2014	2.979069233
mys	Malaysia	2014	2.830211163
arm	Armenia	2014	0.657837391
sgp	Singapore	2014	6.149459839
ner	Niger	2014	3.852665424
tha	Thailand	2014	1.301265836
bol	Bolivia	2014	2.777143478
pak	Pakistan	2014	5.799799442
cog	Congo, Rep.	2014	1.985445976
hnd	Honduras	2014	3.408700943
khm	Cambodia	2014	13.29455566
mus	Mauritius	2014	0.864331424
nga	Nigeria	2014	2.294961452
mex	Mexico	2014	3.424029827
bra	Brazil	2014	0.857745588
col	Colombia	2014	2.212441921
phl	Philippines	2014	4.225873947
irq	Iraq	2014	..
mrt	Mauritania	2014	6.480395794
nic	Nicaragua	2014	1.053774357
dom	Dominican Republic	2014	2.31805563
slv	El Salvador	2014	4.556787968
pan	Panama	2014	1.557950377
vnm	Vietnam	2014	0.496506929
idn	Indonesia	2014	0.449631453
kgz	Kyrgyz Republic	2014	..
rou	Romania	2014	0.462934017
ecu	Ecuador	2014	..
per	Peru	2014	0
lka	Sri Lanka	2014	0.076636843
arg	Argentina	2014	0.432319582
alb	Albania	2014	..
geo	Georgia	2014	..
gtm	Guatemala	2014	1.770276785
tun	Tunisia	2014	0.61870122
ind	India	2014	2.352210045
egy	Egypt	2014	1.144944191
jor	Jordan	2014	0.468665838
afg	Afghanistan	2014	0.304403812
mmr	Myanmar	2014	0.163792059
mar	Morocco	2014	..
eth	Ethiopia	2014	0.027894864
lbr	Liberia	2017	20.84339333
mwi	Malawi	2017	20.30283165
zaf	South Africa	2017	19.0168457
chl	Chile	2017	18.67081451
ben	Benin	2017	18.09383774
tur	Turkey	2017	16.41195297
cod	Congo, Dem. Rep.	2017	16.09634018
tcd	Chad	2017	15.23340988
cmr	Cameroon	2017	15.13783073
gin	Guinea	2017	13.78046894
hti	Haiti	2017	13.51718903
mdg	Madagascar	2017	12.05530262
sle	Sierra Leone	2017	11.01861382
ven	Venezuela	2017	10.97038269
mys	Malaysia	2017	10.87633324
arm	Armenia	2017	9.755306244
sgp	Singapore	2017	9.548007965
ner	Niger	2017	8.717238426
tha	Thailand	2017	8.256859779
bol	Bolivia	2017	7.119389534
pak	Pakistan	2017	6.885557652
cog	Congo, Rep.	2017	6.238439083
hnd	Honduras	2017	6.196012497
khm	Cambodia	2017	5.659508705
mus	Mauritius	2017	5.626936913
nga	Nigeria	2017	5.606164932
mex	Mexico	2017	5.552579403
bra	Brazil	2017	4.836530685
col	Colombia	2017	4.737451077
phl	Philippines	2017	4.521298885
irq	Iraq	2017	4.214755535
mrt	Mauritania	2017	4.012377739
nic	Nicaragua	2017	3.927416801
dom	Dominican Republic	2017	3.894679546
slv	El Salvador	2017	3.549614906
pan	Panama	2017	3.503168821
vnm	Vietnam	2017	3.499500751
idn	Indonesia	2017	3.123915672
kgz	Kyrgyz Republic	2017	3.106876373
rou	Romania	2017	2.990169525
ecu	Ecuador	2017	2.936974764
per	Peru	2017	2.634166241
lka	Sri Lanka	2017	2.420760155
arg	Argentina	2017	2.416167498
alb	Albania	2017	2.377762079
geo	Georgia	2017	2.201021671
gtm	Guatemala	2017	2.126405954
tun	Tunisia	2017	2.037189722
ind	India	2017	1.994543195
egy	Egypt	2017	1.792016268
jor	Jordan	2017	1.060968399
afg	Afghanistan	2017	0.913816154
mmr	Myanmar	2017	0.690499902
mar	Morocco	2017	0.639211476
eth	Ethiopia	2017	0.315928906