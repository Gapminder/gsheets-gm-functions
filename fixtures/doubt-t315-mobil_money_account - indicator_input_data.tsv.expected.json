[
  [
    "geo",
    "name",
    "year",
    "value\r"
  ],
  [
    "lbr",
    "Liberia",
    2014,
    null
  ],
  [
    "lbr",
    "Liberia",
    2015,
    0
  ],
  [
    "lbr",
    "Liberia",
    2016,
    0
  ],
  [
    "lbr",
    "Liberia",
    2017,
    20.84339333
  ],
  [
    "mwi",
    "Malawi",
    2014,
    3.844100952
  ],
  [
    "mwi",
    "Malawi",
    2015,
    0
  ],
  [
    "mwi",
    "Malawi",
    2016,
    0
  ],
  [
    "mwi",
    "Malawi",
    2017,
    20.30283165
  ],
  [
    "zaf",
    "South Africa",
    2014,
    14.42818069
  ],
  [
    "zaf",
    "South Africa",
    2015,
    0
  ],
  [
    "zaf",
    "South Africa",
    2016,
    0
  ],
  [
    "zaf",
    "South Africa",
    2017,
    19.0168457
  ],
  [
    "chl",
    "Chile",
    2014,
    3.789850712
  ],
  [
    "chl",
    "Chile",
    2015,
    0
  ],
  [
    "chl",
    "Chile",
    2016,
    0
  ],
  [
    "chl",
    "Chile",
    2017,
    18.67081451
  ],
  [
    "ben",
    "Benin",
    2014,
    2.022727013
  ],
  [
    "ben",
    "Benin",
    2015,
    0
  ],
  [
    "ben",
    "Benin",
    2016,
    0
  ],
  [
    "ben",
    "Benin",
    2017,
    18.09383774
  ],
  [
    "tur",
    "Turkey",
    2014,
    0.808199763
  ],
  [
    "tur",
    "Turkey",
    2015,
    0
  ],
  [
    "tur",
    "Turkey",
    2016,
    0
  ],
  [
    "tur",
    "Turkey",
    2017,
    16.41195297
  ],
  [
    "cod",
    "Congo, Dem. Rep.",
    2014,
    9.20866394
  ],
  [
    "cod",
    "Congo, Dem. Rep.",
    2015,
    0
  ],
  [
    "cod",
    "Congo, Dem. Rep.",
    2016,
    0
  ],
  [
    "cod",
    "Congo, Dem. Rep.",
    2017,
    16.09634018
  ],
  [
    "tcd",
    "Chad",
    2014,
    5.750106812
  ],
  [
    "tcd",
    "Chad",
    2015,
    0
  ],
  [
    "tcd",
    "Chad",
    2016,
    0
  ],
  [
    "tcd",
    "Chad",
    2017,
    15.23340988
  ],
  [
    "cmr",
    "Cameroon",
    2014,
    1.796604872
  ],
  [
    "cmr",
    "Cameroon",
    2015,
    0
  ],
  [
    "cmr",
    "Cameroon",
    2016,
    0
  ],
  [
    "cmr",
    "Cameroon",
    2017,
    15.13783073
  ],
  [
    "gin",
    "Guinea",
    2014,
    1.473285675
  ],
  [
    "gin",
    "Guinea",
    2015,
    0
  ],
  [
    "gin",
    "Guinea",
    2016,
    0
  ],
  [
    "gin",
    "Guinea",
    2017,
    13.78046894
  ],
  [
    "hti",
    "Haiti",
    2014,
    3.782689571
  ],
  [
    "hti",
    "Haiti",
    2015,
    0
  ],
  [
    "hti",
    "Haiti",
    2016,
    0
  ],
  [
    "hti",
    "Haiti",
    2017,
    13.51718903
  ],
  [
    "mdg",
    "Madagascar",
    2014,
    4.35601902
  ],
  [
    "mdg",
    "Madagascar",
    2015,
    0
  ],
  [
    "mdg",
    "Madagascar",
    2016,
    0
  ],
  [
    "mdg",
    "Madagascar",
    2017,
    12.05530262
  ],
  [
    "sle",
    "Sierra Leone",
    2014,
    4.45238781
  ],
  [
    "sle",
    "Sierra Leone",
    2015,
    0
  ],
  [
    "sle",
    "Sierra Leone",
    2016,
    0
  ],
  [
    "sle",
    "Sierra Leone",
    2017,
    11.01861382
  ],
  [
    "ven",
    "Venezuela",
    2014,
    2.979069233
  ],
  [
    "ven",
    "Venezuela",
    2015,
    0
  ],
  [
    "ven",
    "Venezuela",
    2016,
    0
  ],
  [
    "ven",
    "Venezuela",
    2017,
    10.97038269
  ],
  [
    "mys",
    "Malaysia",
    2014,
    2.830211163
  ],
  [
    "mys",
    "Malaysia",
    2015,
    0
  ],
  [
    "mys",
    "Malaysia",
    2016,
    0
  ],
  [
    "mys",
    "Malaysia",
    2017,
    10.87633324
  ],
  [
    "arm",
    "Armenia",
    2014,
    0.657837391
  ],
  [
    "arm",
    "Armenia",
    2015,
    0
  ],
  [
    "arm",
    "Armenia",
    2016,
    0
  ],
  [
    "arm",
    "Armenia",
    2017,
    9.755306244
  ],
  [
    "sgp",
    "Singapore",
    2014,
    6.149459839
  ],
  [
    "sgp",
    "Singapore",
    2015,
    0
  ],
  [
    "sgp",
    "Singapore",
    2016,
    0
  ],
  [
    "sgp",
    "Singapore",
    2017,
    9.548007965
  ],
  [
    "ner",
    "Niger",
    2014,
    3.852665424
  ],
  [
    "ner",
    "Niger",
    2015,
    0
  ],
  [
    "ner",
    "Niger",
    2016,
    0
  ],
  [
    "ner",
    "Niger",
    2017,
    8.717238426
  ],
  [
    "tha",
    "Thailand",
    2014,
    1.301265836
  ],
  [
    "tha",
    "Thailand",
    2015,
    0
  ],
  [
    "tha",
    "Thailand",
    2016,
    0
  ],
  [
    "tha",
    "Thailand",
    2017,
    8.256859779
  ],
  [
    "bol",
    "Bolivia",
    2014,
    2.777143478
  ],
  [
    "bol",
    "Bolivia",
    2015,
    0
  ],
  [
    "bol",
    "Bolivia",
    2016,
    0
  ],
  [
    "bol",
    "Bolivia",
    2017,
    7.119389534
  ],
  [
    "pak",
    "Pakistan",
    2014,
    5.799799442
  ],
  [
    "pak",
    "Pakistan",
    2015,
    0
  ],
  [
    "pak",
    "Pakistan",
    2016,
    0
  ],
  [
    "pak",
    "Pakistan",
    2017,
    6.885557652
  ],
  [
    "cog",
    "Congo, Rep.",
    2014,
    1.985445976
  ],
  [
    "cog",
    "Congo, Rep.",
    2015,
    0
  ],
  [
    "cog",
    "Congo, Rep.",
    2016,
    0
  ],
  [
    "cog",
    "Congo, Rep.",
    2017,
    6.238439083
  ],
  [
    "hnd",
    "Honduras",
    2014,
    3.408700943
  ],
  [
    "hnd",
    "Honduras",
    2015,
    0
  ],
  [
    "hnd",
    "Honduras",
    2016,
    0
  ],
  [
    "hnd",
    "Honduras",
    2017,
    6.196012497
  ],
  [
    "khm",
    "Cambodia",
    2014,
    13.29455566
  ],
  [
    "khm",
    "Cambodia",
    2015,
    0
  ],
  [
    "khm",
    "Cambodia",
    2016,
    0
  ],
  [
    "khm",
    "Cambodia",
    2017,
    5.659508705
  ],
  [
    "mus",
    "Mauritius",
    2014,
    0.864331424
  ],
  [
    "mus",
    "Mauritius",
    2015,
    0
  ],
  [
    "mus",
    "Mauritius",
    2016,
    0
  ],
  [
    "mus",
    "Mauritius",
    2017,
    5.626936913
  ],
  [
    "nga",
    "Nigeria",
    2014,
    2.294961452
  ],
  [
    "nga",
    "Nigeria",
    2015,
    0
  ],
  [
    "nga",
    "Nigeria",
    2016,
    0
  ],
  [
    "nga",
    "Nigeria",
    2017,
    5.606164932
  ],
  [
    "mex",
    "Mexico",
    2014,
    3.424029827
  ],
  [
    "mex",
    "Mexico",
    2015,
    0
  ],
  [
    "mex",
    "Mexico",
    2016,
    0
  ],
  [
    "mex",
    "Mexico",
    2017,
    5.552579403
  ],
  [
    "bra",
    "Brazil",
    2014,
    0.857745588
  ],
  [
    "bra",
    "Brazil",
    2015,
    0
  ],
  [
    "bra",
    "Brazil",
    2016,
    0
  ],
  [
    "bra",
    "Brazil",
    2017,
    4.836530685
  ],
  [
    "col",
    "Colombia",
    2014,
    2.212441921
  ],
  [
    "col",
    "Colombia",
    2015,
    0
  ],
  [
    "col",
    "Colombia",
    2016,
    0
  ],
  [
    "col",
    "Colombia",
    2017,
    4.737451077
  ],
  [
    "phl",
    "Philippines",
    2014,
    4.225873947
  ],
  [
    "phl",
    "Philippines",
    2015,
    0
  ],
  [
    "phl",
    "Philippines",
    2016,
    0
  ],
  [
    "phl",
    "Philippines",
    2017,
    4.521298885
  ],
  [
    "irq",
    "Iraq",
    2014,
    null
  ],
  [
    "irq",
    "Iraq",
    2015,
    0
  ],
  [
    "irq",
    "Iraq",
    2016,
    0
  ],
  [
    "irq",
    "Iraq",
    2017,
    4.214755535
  ],
  [
    "mrt",
    "Mauritania",
    2014,
    6.480395794
  ],
  [
    "mrt",
    "Mauritania",
    2015,
    0
  ],
  [
    "mrt",
    "Mauritania",
    2016,
    0
  ],
  [
    "mrt",
    "Mauritania",
    2017,
    4.012377739
  ],
  [
    "nic",
    "Nicaragua",
    2014,
    1.053774357
  ],
  [
    "nic",
    "Nicaragua",
    2015,
    0
  ],
  [
    "nic",
    "Nicaragua",
    2016,
    0
  ],
  [
    "nic",
    "Nicaragua",
    2017,
    3.927416801
  ],
  [
    "dom",
    "Dominican Republic",
    2014,
    2.31805563
  ],
  [
    "dom",
    "Dominican Republic",
    2015,
    0
  ],
  [
    "dom",
    "Dominican Republic",
    2016,
    0
  ],
  [
    "dom",
    "Dominican Republic",
    2017,
    3.894679546
  ],
  [
    "slv",
    "El Salvador",
    2014,
    4.556787968
  ],
  [
    "slv",
    "El Salvador",
    2015,
    0
  ],
  [
    "slv",
    "El Salvador",
    2016,
    0
  ],
  [
    "slv",
    "El Salvador",
    2017,
    3.549614906
  ],
  [
    "pan",
    "Panama",
    2014,
    1.557950377
  ],
  [
    "pan",
    "Panama",
    2015,
    0
  ],
  [
    "pan",
    "Panama",
    2016,
    0
  ],
  [
    "pan",
    "Panama",
    2017,
    3.503168821
  ],
  [
    "vnm",
    "Vietnam",
    2014,
    0.496506929
  ],
  [
    "vnm",
    "Vietnam",
    2015,
    0
  ],
  [
    "vnm",
    "Vietnam",
    2016,
    0
  ],
  [
    "vnm",
    "Vietnam",
    2017,
    3.499500751
  ],
  [
    "idn",
    "Indonesia",
    2014,
    0.449631453
  ],
  [
    "idn",
    "Indonesia",
    2015,
    0
  ],
  [
    "idn",
    "Indonesia",
    2016,
    0
  ],
  [
    "idn",
    "Indonesia",
    2017,
    3.123915672
  ],
  [
    "kgz",
    "Kyrgyz Republic",
    2014,
    null
  ],
  [
    "kgz",
    "Kyrgyz Republic",
    2015,
    0
  ],
  [
    "kgz",
    "Kyrgyz Republic",
    2016,
    0
  ],
  [
    "kgz",
    "Kyrgyz Republic",
    2017,
    3.106876373
  ],
  [
    "rou",
    "Romania",
    2014,
    0.462934017
  ],
  [
    "rou",
    "Romania",
    2015,
    0
  ],
  [
    "rou",
    "Romania",
    2016,
    0
  ],
  [
    "rou",
    "Romania",
    2017,
    2.990169525
  ],
  [
    "ecu",
    "Ecuador",
    2014,
    null
  ],
  [
    "ecu",
    "Ecuador",
    2015,
    0
  ],
  [
    "ecu",
    "Ecuador",
    2016,
    0
  ],
  [
    "ecu",
    "Ecuador",
    2017,
    2.936974764
  ],
  [
    "per",
    "Peru",
    2014,
    0
  ],
  [
    "per",
    "Peru",
    2015,
    0
  ],
  [
    "per",
    "Peru",
    2016,
    0
  ],
  [
    "per",
    "Peru",
    2017,
    2.634166241
  ],
  [
    "lka",
    "Sri Lanka",
    2014,
    0.076636843
  ],
  [
    "lka",
    "Sri Lanka",
    2015,
    0
  ],
  [
    "lka",
    "Sri Lanka",
    2016,
    0
  ],
  [
    "lka",
    "Sri Lanka",
    2017,
    2.420760155
  ],
  [
    "arg",
    "Argentina",
    2014,
    0.432319582
  ],
  [
    "arg",
    "Argentina",
    2015,
    0
  ],
  [
    "arg",
    "Argentina",
    2016,
    0
  ],
  [
    "arg",
    "Argentina",
    2017,
    2.416167498
  ],
  [
    "alb",
    "Albania",
    2014,
    null
  ],
  [
    "alb",
    "Albania",
    2015,
    0
  ],
  [
    "alb",
    "Albania",
    2016,
    0
  ],
  [
    "alb",
    "Albania",
    2017,
    2.377762079
  ],
  [
    "geo",
    "Georgia",
    2014,
    null
  ],
  [
    "geo",
    "Georgia",
    2015,
    0
  ],
  [
    "geo",
    "Georgia",
    2016,
    0
  ],
  [
    "geo",
    "Georgia",
    2017,
    2.201021671
  ],
  [
    "gtm",
    "Guatemala",
    2014,
    1.770276785
  ],
  [
    "gtm",
    "Guatemala",
    2015,
    0
  ],
  [
    "gtm",
    "Guatemala",
    2016,
    0
  ],
  [
    "gtm",
    "Guatemala",
    2017,
    2.126405954
  ],
  [
    "tun",
    "Tunisia",
    2014,
    0.61870122
  ],
  [
    "tun",
    "Tunisia",
    2015,
    0
  ],
  [
    "tun",
    "Tunisia",
    2016,
    0
  ],
  [
    "tun",
    "Tunisia",
    2017,
    2.037189722
  ],
  [
    "ind",
    "India",
    2014,
    2.352210045
  ],
  [
    "ind",
    "India",
    2015,
    0
  ],
  [
    "ind",
    "India",
    2016,
    0
  ],
  [
    "ind",
    "India",
    2017,
    1.994543195
  ],
  [
    "egy",
    "Egypt",
    2014,
    1.144944191
  ],
  [
    "egy",
    "Egypt",
    2015,
    0
  ],
  [
    "egy",
    "Egypt",
    2016,
    0
  ],
  [
    "egy",
    "Egypt",
    2017,
    1.792016268
  ],
  [
    "jor",
    "Jordan",
    2014,
    0.468665838
  ],
  [
    "jor",
    "Jordan",
    2015,
    0
  ],
  [
    "jor",
    "Jordan",
    2016,
    0
  ],
  [
    "jor",
    "Jordan",
    2017,
    1.060968399
  ],
  [
    "afg",
    "Afghanistan",
    2014,
    0.304403812
  ],
  [
    "afg",
    "Afghanistan",
    2015,
    0
  ],
  [
    "afg",
    "Afghanistan",
    2016,
    0
  ],
  [
    "afg",
    "Afghanistan",
    2017,
    0.913816154
  ],
  [
    "mmr",
    "Myanmar",
    2014,
    0.163792059
  ],
  [
    "mmr",
    "Myanmar",
    2015,
    0
  ],
  [
    "mmr",
    "Myanmar",
    2016,
    0
  ],
  [
    "mmr",
    "Myanmar",
    2017,
    0.690499902
  ],
  [
    "mar",
    "Morocco",
    2014,
    null
  ],
  [
    "mar",
    "Morocco",
    2015,
    0
  ],
  [
    "mar",
    "Morocco",
    2016,
    0
  ],
  [
    "mar",
    "Morocco",
    2017,
    0.639211476
  ],
  [
    "eth",
    "Ethiopia",
    2014,
    0.027894864
  ],
  [
    "eth",
    "Ethiopia",
    2015,
    0
  ],
  [
    "eth",
    "Ethiopia",
    2016,
    0
  ],
  [
    "eth",
    "Ethiopia",
    2017,
    0.315928906
  ]
]